cogs5e/models/automation/effects/damage.py

```diff
@@ -112,6 +113,10 @@ def run(self, autoctx):
         # magic arg (#853), magical effect (#1063)
         magical_effect = autoctx.combatant and autoctx.combatant.active_effects('magical')
         always = {'magical'} if (magical_effect or autoctx.is_spell or magic_arg) else None
+        # silvered arg(#825)
+        silvered_effect = autoctx.combatant and autoctx.combatant.active_effects('silvered')
+        if silvered_effect or silvered_arg:
+            always = {'magical', 'silvered'} if always else {'silvered'}  
```

Comment:
I'd refactor how `always` is handled here to add to the set on a conditional, instead of creating new sets each time - for example (pseudocode):
```
always = empty set
if magical:
  always.add(magical)
if silvered:
  always.add(silvered)
```

Reply:
I thought about it but didn't want to touch too much that was unrelated before making the initial PR, I'll add it in with the tests.